### ISSUE TITLE
Move containerrun out of cf-operator

### ISSUE THREAD

**Issue by mook-as** [cloudfoundry-incubator/kubecf#1048]
This is a proxy card to track the work I'm doing in [#172048264](https://www.pivotaltracker.com/story/show/172048264), mostly to ensure things don't get lost.

Acceptance Criteria:
- [x] we need a [new repo](https://github.com/cloudfoundry-incubator/quarks-container-run/) in the cloudfoundry-incubator extension.
- [ ] should be moved to the stemcell image.
- [ ] remove code that copies this binary around.

**Comment by mook-as** [comment]
Currently waiting on [SR#816074](https://build.opensuse.org/request/show/816074) (landing the package in the stemcell).

**Comment by mook-as** [comment]
This is _not_ blocked, unless you mean "blocked on somebody doing a review"

**Comment by aduffeck** [comment]
[SR#816074](https://build.opensuse.org/request/show/816074) was now accepted.

**Comment by andreas-kupries** [comment]
Blocked for the moment by #941

**Comment by fargozhu** [comment]
Unblocked with #941 closed

**Comment by manno** [comment]
If someone builts a RPM from the separate repo, we can include that in the stemcell at https://build.opensuse.org/project/show/Cloud:Platform:Fissile-Stemcell

Quarks can then queue another story to remove the copy code, remove the build code from Quarks Dockerfiles and so on.

**Comment by mook-as** [comment]
Oh, sorry, should have more context here.
@manno It's already in the stemcell; landed via [SR#816074](https://build.opensuse.org/request/show/816074).  All the images that kubecf currently uses (since the 13.9 bump) has a `/usr/bin/quarks-container-run` binary.  I was going to work on making the PR for quarks-operator to drop the copying, but things got in the way.

**Comment by mook-as** [comment]
It looks like https://github.com/cloudfoundry-incubator/quarks-operator/pull/1074 would handle the actual operator side (that I kept not getting to because other ~shiny~ things kept getting into the way.  Ooops.

**Comment by manno** [comment]
I'm finishing your item in the Quarks backlog and queue a new story for Quarks to make the tests pass. 
https://www.pivotaltracker.com/story/show/172048264

**Comment by viovanov** [comment]
@manno @mook-as this item seems to be done
we're tracking removal of the operator containerrun on the quarks board
